les/peer.go

```diff
@@ -117,8 +115,9 @@ func (m keyValueMap) get(key string, val interface{}) error {
 	return rlp.DecodeBytes(enc, val)
 }
 
-// peerCommons contains fields needed by both server peer and client peer.
-type peerCommons struct {
+// peer represents each node to which the client is connected.
+// The node here refers to the les server.
+type peer struct {
```

Comment:
Any particular reason for merging `clientPeer` and `serverPeer` together? It would be nice to merge them, but these two peers have a lot of different fields, feel a bit messy to put all fields together.

But I guess there are some reasons behind this change.

Reply:
If we really want to merge these two together, then maybe we can have the nested structs called `clientFields`, `serverFields` in the `peer` to include the fields for clients and servers.

e.g.

```go

type peer struct {
     commonField a
            ...
     commonField z

     serverFields fields
     clientFields fields
} 
```

Maybe it can make the fields cleaner for their purposes?